packages/venia-ui/lib/components/AccountMenu/accountMenu.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { bool, shape, string } from 'prop-types';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
```

Comment:
I have seen this in a couple of places. Are we fine with importing a file in the same package using the package namespace?

Reply:
We actually prefer it for DX because when developers are modifying our components the relative paths become invalid.

See the [Modify Footer Component](https://magento.github.io/pwa-studio/tutorials/pwa-studio-fundamentals/modify-site-footer/#update-footer-import-statements) tutorial on our docs site. We can eliminate this step for developers by doing non-relative imports up front.

Reply:
Good to know. But, I remember @zetlen mentioned something about this pattern. I remember it to be something on the negative side rather than positive.